OpenRA.Mods.Common/Traits/Player/ResourceDisplay.cs

```diff
@@ -0,0 +1,78 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public class ResourceDisplayInfo : ITraitInfo, Requires<PlayerResourcesInfo>
+	{
+		public object Create(ActorInitializer init) { return new ResourceDisplay(init.Self, this); }
+	}
+
+	public class ResourceDisplay : ITick, IResourceDisplay
```

Comment:
It occurs to me that this doesn't even need to be a trait.  This is all logic for the chrome, so it would better live in a ChromeLogic class (e.g. `IngameCashCounterLogic`).